src/Provider/FileProvider.php

```diff
@@ -250,8 +250,11 @@ public function getDownloadResponse(MediaInterface $media, string $format, strin
         );
     }
 
-    public function validate(ErrorElement $errorElement, MediaInterface $media): void
+    public function validateMedia(ExecutionContextInterface $context, MediaInterface $media): void
```

Comment:
There is no BC break issue if someone is extending the validate method ?

Reply:
It should be compatible as I added those checks if a child class overwrites them inside `BaseProvider` :thinking: